source/databricks/calculation_engine/package/settlement_report_job/table_column_names.py

```diff
@@ -0,0 +1,22 @@
+class TableColumnNames:
```

Comment:
We should be reading from the standard data product. So it's more likely view column names. But I'm not sure that that will lead us to the best name here. Perhaps like this...?
```suggestion
class DataProductColumnNames:
```